catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsResponseExceptionMapper.java

```diff
@@ -48,12 +52,12 @@ public WfsException fromResponse(Response response) {
           is.reset();
           msg = IOUtils.toString(is);
         } catch (IOException e) {
-          wfsEx = new WfsException("Error reading Response" + (msg != null ? ": " + msg : ""), e);
+          LOGGER.info("Unable to parse exception report: {}", e.getMessage());
```

Comment:
✏️  By delaying the creation of the exception below, we are now loosing the cause for the exception which was passed to the ctor here